global/scripts/debug.shlib

```diff
@@ -0,0 +1,262 @@
+#!/bin/echo This script should not run directly:
+
+# Description:
+#
+#   Function for printing debugging information on caught errors.
+#   It makes use of error trapping. To reset the trap, add
+#   end of the script using "trap - ERR".
+#
+# Input:
+#
+#   - line number
+#   - command ran
+#   - exit status
```

Comment:
Can you explicitly, as part of the input usage indicate that:
```
$1 - line number
$2 - command run
$3 - exit status
```

Reply:
Added.